audit/projects/k8s-conform/buckets/k8s-conform-inspur/iam.json

```diff
@@ -16,7 +16,16 @@
     },
     {
       "members": [
-        "group:k8s-infra-artifact-admins@kubernetes.io"
+        "group:k8s-infra-conform-inspur@kubernetes.io",
+        "serviceAccount:service-inspur@k8s-conform.iam.gserviceaccount.com"
```

Comment:
This was improperly setup before, ran the appropriate script during troubleshooting https://github.com/kubernetes/test-infra/issues/20914#issuecomment-783948420